projects/core/src/lib/mask.ts

```diff
@@ -318,14 +318,12 @@ export class Maskito extends MaskHistory {
                 initialElementState.value.slice(to),
             selection: [from + data.length, from + data.length],
```

Comment:
why not 

```ts
const newPossibleState: ElementState = this.clampState({
            value:
                initialState.value.slice(0, initialFrom) +
                initialState.value.slice(initialTo),
            selection: [initialFrom, initialFrom],
        });
```

Reply:
Variable `newPossibleState` is used twice in this method.
The last usage:
```js
newPossibleState.value.length > maxLength
```

We should use **un**clampled version for this check.